Resources/public/js/views/services/ezsearch-searchviewservice.js

```diff
@@ -2,21 +2,53 @@ YUI.add('ezsearch-searchviewservice', function (Y) {
     Y.namespace('eZSearch');
 
     Y.eZSearch.SearchViewService = Y.Base.create('ezsearchSearchViewService', Y.eZ.ViewService, [], {
+        _initializer: function() {
+        },
 
         _load: function (callback) {
             this.set('searchString', this.get('request').params.searchString);
-            callback();
+
+            if (this.get('searchString')) {
+                this.fire('locationSearch', {
+                    viewName: 'search-' + this.get('searchString'),
+                    resultAttribute: 'searchResultList',
+                    resultTotalCountAttribute: 'searchResultCount',
+                    loadContent: true,
```

Comment:
as far as I can see, we don't need the Content, the ContentInfo available in the Location should be enough